src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,360 @@
+## List folder contents
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
+
+{{site.data.alerts.callout_success}}
+To discover the entire folder structure, you must list each folder's contents separately.
+{{site.data.alerts.end}}
+
+1. To list the clusters and folders at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+           --url 'https://management-staging.crdb.io/api/v1/folders/root/contents' \
+           --header 'Authorization: Bearer <api_key>'
+    ~~~
+
+    Replace `<api-key>` with your service account's API key.
+
+1. To list the clusters and folders contained within another folder, specify its ID instead of `root`:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+           --url 'https://management-staging.crdb.io/api/v1/folders/<folder_id>/contents' \
+           --header 'Authorization: Bearer <api_key>'
+    ~~~
+
+    Replace:
+    - `<folder-id>`: The ID of a folder.
+    - `<api-key>`: Your service account's API key.
+
+{{site.data.alerts.callout_info}}
+In the API, each folder is referred to by its ID, regardless of where it is located in the hierarchy.
+{{site.data.alerts.end}}
+
+## Create a cluster in a folder
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Cluster Administrator](authorization.html#cluster-administrator) or [Cluster Creator](authorization.html#cluster-creator)
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
+
+To create a cluster in a folder, set the `parent_id` field to the ID of the folder.
+
+{{site.data.alerts.callout_info}}
+If you do not specify a `parent_id`, or if you set `parent_id` to `root`, the cluster is created at the level of the organization.
+{{site.data.alerts.end}}
+
+For example, to create a {{ site.data.products.serverless }} deployed on GCP in a folder:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+       --url 'https://management-staging.crdb.io/api/v1/clusters' \
+       --header 'Authorization: Bearer <api-key>' \
+       --header 'content-type: application/json' \
+       --data '{"name":"<cluster_name>","provider":"GCP","spec":{"serverless":{"regions":["us-central1"],"spend_limit":0}, "parent_id":"<parent_id>"}}'
+~~~
+
+Replace:
+
+- `<cluster_name>`: A name for the cluster. Replace `<parent_id>` with the ID of the folder.
+- `<api-key>`: Your service account's API key.
+- `<parent-id>`: The ID of a folder.
+
+## Move a cluster into or out of a folder
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
```

Comment:
@mdlinville they need Folder Mover in order to move the Folder. I believe Folder Admin can't move them but @jason-crl @cameronnunez can confirm.

Reply:
Also, they need to have the role at the destination and the source folders.

Reply:
Done